### PR TITLE
Cust 2140 sidenav theme

### PR DESCRIPTION
**Pull request by jsmylnycky**
Restructured sidenav for new standards
Fixed config from ending up in the mobileApps array instead of the coreAppsBottom
Fixed flex layout of peachSelect
Fixed bootstrapAccount to look for core prefs for users and accounts instead of the currently active app

Note: I did have to merge with master so theres some extraneous code in here.  Sorry!

### CODE REVIEW

**Comment by rynomars** [comment]
:thumbsup:

File: src/app/run/run.js [hunk 1]
```diff
@@ -3,7 +3,7 @@
 function AppRun($rootScope, $peach, $location, $log) {
   $rootScope.states = $peach.session.states;
 
-  Offline.options = {checks: {xhr: {url: '/health-check'}}};
+  //Offline.options = { checks: { xhr: { url: '/health-check' } } };
```
Comment (rwrzesien) [inline src/app/run/run.js]: Why keeping commented code ?

**Comment by rwrzesien** [comment]
Besides my one comment :+1:

**Comment by darushka** [comment]
:+1:

File: src/app/run/run.js (hunk 1 above)
Comment (jsmylnycky) [inline src/app/run/run.js]: Im doing some continued improvements on the Offline plugin and I dont want to lose how that was setup, without having to dig through old commits to find it again.

**Comment by jsmylnycky** [comment]
Released v0.6.0

**Pull request closed without merging**